feat(serverless): add more entries in faq/troubleshooting (#3477)
scaleway/docs-content: serverless/functions/troubleshooting/common-errors.mdx

- Update your function's resources by provisioning more memory and more vCPU.

## Function in error state

### Cause

> This error is returned when the Function can't start

Reviewer: ```suggestion
This error is returned when the Function cannot start
```

> This error is returned when the Function cannot start due to insufficient provisioned resources.

Reviewer 2: ```suggestion
This error is returned when the function cannot start due to insufficient provisioned resources.
```

### Possible solutions

> Functions can't start when not enough resources are set. Try increasing vCPU and RAM.

Reviewer: ```suggestion
Functions cannot start when not enough resources are set. Try increasing vCPU and RAM.
```

> ### Cause
> 
> This error is returned when the Function can't start
> 
> ### Possible solutions
> 
> Functions can't start when not enough resources are set. Try increasing vCPU and RAM.
> 
> ## Timeout error

Reviewer 3: ```suggestion
### Cause

This error is returned when the Function cannot start due to insufficient provisioned resources.

### Possible solutions

Update your function's resources by provisioning more memory and more vCPU.

## Timeout error
```
